src/views/splash/splash.jsx

```diff
@@ -448,10 +223,15 @@ var Splash = injectIntl(React.createClass({
 var mapStateToProps = function (state) {
     return {
         session: state.session,
-        permissions: state.permissions
+        permissions: state.permissions,
+        rows: state.rows,
+        activity: state.activity,
+        news: state.news,
+        projectCount: state.projectCount,
+        homepageCache: state.homepageCache
```

Comment:
I'd rather it be part of this PR since this PR adds a lot of redux state as props.